src/bumerang/borrowhandler.py

```diff
@@ -58,8 +58,8 @@ def _create_borrow_node(self):
             into the database
         """
         return {
-            'title': self.get_argument('title'),
-            'description': self.get_argument('description', ''),
-            'distance': self.get_argument('distance'),
-            'duration': self.get_argument('duration')
+            'title': self.request.arguments['title'],
+            'description': self.request.arguments['description'],
```

Comment:
this makes the description mandatory, if we want to make this optional, use `self.request.arguments.get('description)`